DeviceTests/DeviceTests.Android/Properties/AndroidManifest.xml

```diff
@@ -1,45 +1,56 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0.1.0" package="com.xamarin.essentials.devicetests" android:installLocation="auto">
-	<uses-sdk android:minSdkVersion="19" android:targetSdkVersion="29" />
-	<uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
-	<uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
-	<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
-	<uses-permission android:name="android.permission.BATTERY_STATS" />
-	<uses-permission android:name="android.permission.CAMERA" />
-    <uses-permission android:name="android.permission.FLASHLIGHT" />
-	<uses-permission android:name="android.permission.INTERNET" />
-	<uses-permission android:name="android.permission.VIBRATE" />
-	<uses-permission android:name="android.permission.READ_CONTACTS" />
-	<queries>
-		<!-- Email -->
-		<intent>
-			<action android:name="android.intent.action.SENDTO" />
-			<data android:scheme="mailto" />
-		</intent>
-		<!-- Browser -->
-		<intent>
-			<action android:name="android.intent.action.VIEW" />
-			<data android:scheme="http" />
-		</intent>
-		<!-- Browser -->
-		<intent>
-			<action android:name="android.intent.action.VIEW" />
-			<data android:scheme="https" />
-		</intent>
-		<!-- Sms -->
-		<intent>
-			<action android:name="android.intent.action.VIEW" />
-			<data android:scheme="smsto" />
-		</intent>
-		<!-- PhoneDialer -->
-		<intent>
-			<action android:name="android.intent.action.DIAL" />
-			<data android:scheme="tel" />
-		</intent>
-		<!-- MediaPicker -->
-		<intent>
-			<action android:name="android.media.action.IMAGE_CAPTURE" />
-		</intent>
-	</queries>
-	<application android:label="@string/app_name" android:icon="@mipmap/ic_launcher" android:roundIcon="@mipmap/ic_launcher_round" android:theme="@style/MainTheme"></application>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0" package="com.xamarin.essentials" android:installLocation="auto">
+  <uses-sdk android:minSdkVersion="21" android:targetSdkVersion="30" />
+  <!--Camera-->
+  <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" android:maxSdkVersion="29"/>
+  <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" android:maxSdkVersion="32"/>
+  <!--Android 13 specific media folders-->
+  <uses-permission android:name="android.permission.READ_MEDIA_AUDIO" android:minSdkVersion="32"/>
```

Comment:
android:minSdkVersion for these READ_MEDIA_* permissions should be 33 since they were only added in Android API 33 (Android 13). See here: https://developer.android.com/reference/android/Manifest.permission#READ_MEDIA_AUDIO